blog/steering/governance-changes.md

```diff
@@ -0,0 +1,87 @@
+process to get to this new charter, and I first and foremost want to say a
+sincere THANK YOU to everyone in our community that participated in our public
+discussions on this topic. The community participation, which included
+individual code and non-code contributors, as well as working group leads,
+helped to influence the direction the discussions went.
+
+Let me summarize the meaning of the changes, and then go into some detail:
+
+- There will be elections for Steering Committee members
+- Steering committee members serve as individuals rather than representing their employer
+- No vendor will be allowed to have a majority of seats in the Steering Committee
+- We added the new Knative Trademark Committee (KTC) to handle matters related to the trademark
+- There is a transition plan from the current bootstrap scheme to the new governance scheme
+
+Those of you who love committees will be pleased that we have increased the
+committee count within the project by one. Those of you who don't love
+committees for their own sake may be asking what the purpose of such a committee
+is; luckily, there is a clear and sensible answer. During the discussions around
+new governance, it became apparent that there was a clear separation of concerns
+between the specific vendor needs around trademark, "core" scope (for lack of a
+better term) and conformance, and the broader community needs around everything
+else Steering does. While we could have created a hybrid structure for Steering,
+we decided it was simpler to just have a vendor committee that focused only on
+scope and conformance, with the Steering committee focusing on the other aspects
+of community growth and project governance.
+
+The KTC will initially include representatives from Google, Red Hat/IBM, and
+VMware. They will need to figure out how new components and features "plug in"
+to the Knative ecosystem in a non-technical sense. They will also help Steering
+settle questions around derivative trademarks and logos, Knative "extensions",
+and other questions about being part of our larger project. Membership to the KTC
+is renewed on an annual basis, at the same time that the 2021 steering elections happen.
+When the committee is renewed, new vendors may receive a seat on the KTC, by
+virtue of their contributions in the previous year. Since we know that not all
+contributions are easy to quantify, any vendor may frame their contributions,
+and request consideration for membership on the KTC.
+
+With regard to the Steering Committee, the five members will be elected by the
+contributors to Knative, the same way that TOC members are. The difference
+between Steering and TOC will be that TOC members need to be code reviewers,
+whereas Steering members might contribute in other areas and will lead the
+project in non-code areas. To ensure fairness, no more than two Steering members
+may work for the same employer. As the project grows, it's possible that we may
```

Comment:
wonder if we should not add this (to 7 seats in the future), but maybe say something about the fact that "as the project and its needs evolve, we will revisit the mechanics of the steering to ensure that the project needs are met". or sumtin like that.